libkbfs/keybase_daemon_rpc.go

```diff
@@ -403,9 +413,31 @@ func (k *KeybaseDaemonRPC) Shutdown() {
 	if k.keepAliveCancel != nil {
 		k.keepAliveCancel()
 	}
+	k.log.Warning("Keybase service shutdown")
+
 }
 
 // TeamExit (does not) implement keybase1.NotifyTeamInterface.
 func (k *KeybaseDaemonRPC) TeamExit(context.Context, keybase1.TeamID) error {
 	return nil
 }
+
+// notifyServiceHandler implements keybase1.NotifyServiceInterface
+type notifyServiceHandler struct {
+	config Config
+	log    logger.Logger
+}
+
+func (s notifyServiceHandler) Shutdown(_ context.Context) error {
+	s.log.Warning("NotifyService: Shutdown")
+	if runtime.GOOS == "windows" {
+		os.Exit(0)
+	}
+	return nil
+}
+
+// newNotifyServiceHandler makes a new NotifyServiceHandler
+func newNotifyServiceHandler(config Config, log logger.Logger) keybase1.NotifyServiceInterface {
```

Comment:
Is there a reason we use this by value and not by reference here and in the methods?

Reply:
No, will change